src/components/Content/Content.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { ReactElement, useState } from "react";
+import { ContentProps, ContentType } from "./ContentTypes";
+
+// Components
+import Button from "../Button";
+import Chip from "../Chip";
+import {
+	MainWrapper,
+	Title,
+	TitleWrapper,
+	ButtonsWrapper,
+	Label,
+	Value,
+	LabelValueWrapper,
+	Paragraph,
+	ChipsWrapper,
+	LabelWrapper,
+	ValueFile,
+	ContentRow,
+	ContentColumn,
+	ParagraphWrapper,
+} from "./Content.styled";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+
+const Content = (props: ContentProps): ReactElement => {
+	const { title, onClickEdit, content, onClickAdd } = props;
+	const [showOrHide, setShowOrHide] = useState(content.length < 3);
```

Comment:
Tip (you don't have to change this), you could leave the variable just as "show" since it's a boolean, show: false means that it's hidden (also I feel it's easier to read).